determined-ai/determined
# chore: update to antd v5.3.3

## Description

I couldn't update some big libraries. Antd is one of them.
Part of [WEB-527]

## Test Plan
- Check if no design regression
- Check if new `Breadcrumb` looks the same as before
  - Some props in `Breadcrumb` became deprecated https://ant.design/components/breadcrumb 
  - Experiment Detail page
  - Dashboard
  - Project Detail page
  - Model/Model Version page




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-527]: https://hpe-aiatscale.atlassian.net/browse/WEB-527?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
